pkg/gadgets/trace/tcpconnect/types/types.go

```diff
@@ -41,3 +41,7 @@ func Base(ev eventtypes.Event) *Event {
 		Event: ev,
 	}
 }
+
+func (ev *Event) GetMountNSID() uint64 {
+	return ev.MountNsID
+}
```

Comment:
Absolutely. Just didn't want to touch more existing code of the gadgets than necessary, even if that lead to those additional lines. Do you propose to change it directly or later on when cleaning up?

Reply:
I would change it directly. It is not a change with a significant impact on the current implementation. cc @mauriciovasquezbernal

Reply:
I think we can change it now, it's a change we all agree with.

Reply:
Should we change the type name from `EventWithMountNsID` to `MountNsID`? Because it's not actually an "Event with".

Reply:
Yes, `MountNsID` sounds better than `EventWithMountNsID`.

Reply:
I proposed `EventWithMountNsID` because, in Go, one usually uses composition when there is an "is-a" relation. While if it is a "has-a" relation, we should add it as a member, and its methods (`GetMountNSID` in this case) wouldn't be promoted. Anyway, it is just a detail.